lib/scala/pkg/src/main/scala/org/enso/pkg/Package.scala

```diff
@@ -307,7 +308,8 @@ class PackageManager[F](implicit val fileSystem: FileSystem[F]) {
     authors: List[Contact]                   = List(),
     maintainers: List[Contact]               = List(),
     license: String                          = "",
-    componentGroups: Option[ComponentGroups] = None
+    componentGroups: Option[ComponentGroups] = None,
+    jvm: Option[Boolean]                     = None
```

Comment:
A tri-state value. Shouldn't we use just `Boolean` - e.g. binary state here?

Reply:
I don't believe so, no.